flashinfer/attention.py

```
@@ -118,9 +158,37 @@ def plan(
+        if self._use_cuda_graph:
+            if batch_size != self._fixed_batch_size:
+                raise ValueError(
+                    "The batch size should be fixed in cudagraph mode, the runtime batch size {} "
+                    " mismatches the batch size set during initialization {}".format(
+                        batch_size, self._fixed_batch_size
+                    )
+                )
+            if len(qo_indptr) != self._fixed_batch_size + 1:
+                raise ValueError(
+                    "The length of qo_indptr should be batch_size + 1 in cudagraph mode"
+                )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)